components/webfield/ProgramChairConsole/PaperStatus.js

```diff
@@ -117,15 +119,15 @@ const PaperRow = ({
           referrerUrl={referrerUrl}
           shortPhrase={shortPhrase}
           submissionName={submissionName}
-          reviewerAssignmentUrl={getManualAssignmentUrl('Reviewers')}
+          reviewerAssignmentUrl={getManualAssignmentUrl(`${prettyField(reviewerName)}`)}
```

Comment:
I don't think you need to do prettyField here because the function is using the param to search the assignment invitation.

Reply:
Fixed